tests/test_gsc_discovery.py

```diff
@@ -1,30 +1,29 @@
 """Test tap discovery mode and metadata."""
 import re
 
-from tap_tester import menagerie, connections
-
 from base import GoogleSearchConsoleBaseTest
+from tap_tester import connections, menagerie
 
-class GoogleSearchConsoleDiscovery(GoogleSearchConsoleBaseTest):
 
+class GoogleSearchConsoleDiscovery(GoogleSearchConsoleBaseTest):
     def name(self):
         return "tap_tester_google_search_console_discovery_test"
 
     def test_run(self):
-        """
-        Testing that discovery creates the appropriate catalog with valid metadata.
-        • Verify number of actual streams discovered match expected
-        • Verify the stream names discovered were what we expect
-        • Verify stream names follow naming convention
-          streams should only have lowercase alphas and underscores
-        • verify there is only 1 top level breadcrumb
-        • verify replication key(s)
-        • verify primary key(s)
-        • verify that if there is a replication key we are doing INCREMENTAL otherwise FULL
-        • verify the actual replication matches our expected replication method
-        • verify that primary, replication and foreign keys
-          are given the inclusion of automatic.
-        • verify that all other fields have inclusion of available metadata.
+        """Testing that discovery creates the appropriate catalog with valid
+        metadata.
+
+        • Verify number of actual streams discovered match expected •
+        Verify the stream names discovered were what we expect • Verify
+        stream names follow naming convention   streams should only have
+        lowercase alphas and underscores • verify there is only 1 top
+        level breadcrumb • verify replication key(s) • verify primary
+        key(s) • verify that if there is a replication key we are doing
+        INCREMENTAL otherwise FULL • verify the actual replication
+        matches our expected replication method • verify that primary,
+        replication and foreign keys   are given the inclusion of
+        automatic. • verify that all other fields have inclusion of
+        available metadata.
```

Comment:
Align it properly, instead of bullets use - and keep each point on a separate line.

Reply:
Done